.circleci/config.yml

```diff
@@ -510,15 +558,33 @@ workflows:
           <<: *defaults
 
       # Transpiler
-      - avm-transpiler:
+      - avm-transpiler-x86_64:
           requires:
-            - noir-ecr-manifest
+            - noir-x86_64
```

Comment:
Since there are now per-arch versions of avm-transpiler, I'm not sure whether this should be ecr-manifest or the version for one arch. Probably doesn't matter?